clearwater-diags-monitor/usr/share/clearwater/bin/gather_diags_and_report_location

```diff
@@ -13,7 +13,10 @@ DIAGS_LOCATION=/var/clearwater-diags-monitor/dumps/
 
 # In some scenarios we are run as root, and $HOME is not defined.
 # Instead, we should create a dump in the Clearwater user's directory.
```

Comment:
Please update this to make it clear that we always want to put diags in the clearwater user's directory, irrespective of whether or not the command was run with `sudo`.